### PR TITLE
add new ui tests for plan selection 

### LINKED ISSUE #1857: Add new ui tests for selecting and viewing subscription plans
As described, add new ui test coverage for selecting and viewing subscription plans

### PR DESCRIPTION
**Pull request by asnaith**
closes #1857

### CODE REVIEW

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PlanDetails.tsx [hunk 1]
```diff
@@ -148,18 +154,21 @@ const PlanDetails = ({ plan, goToSelectPlan, onSelectPlanPrice }: IPlanDetails)
...
-                right={{ value: "monthly" }}
-                onChange={() => setBillingPeriod(billingPeriod === "monthly" ? "yearly" : "monthly")}
-              />
+              <span data-cy="toggle-switch-duration">
+                <ToggleSwitch
+                  left={{ value: "yearly" }}
+                  right={{ value: "monthly" }}
+                  onChange={() => setBillingPeriod(billingPeriod === "monthly" ? "yearly" : "monthly")}
+                />
+              </span>
```
Comment (asnaith) [inline packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PlanDetails.tsx]: The span was necessary here as it did not work when put on the div or toggle switch

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ChangePlanModal.tsx [hunk 2]
```diff
@@ -92,8 +92,12 @@ const ChangeProductModal = ({ onClose }: IChangeProductModal) => {
       injectedClass={{
         inner: classes.inner
       }}
-      testId="change-product"
       onClose={onClose}
+      testId={
+        (slide === "select")
+          ? "select-plan"
+          : (slide === "downgradeDetails" ? "downgrade-details" : "plan-details")
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ChangePlanModal.tsx]:
```suggestion
          : slide === "downgradeDetails"
            ? "downgrade-details"
            : "plan-details"
```

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PlanDetails.tsx (hunk 1 above)
Comment (Tbaut) [reply packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PlanDetails.tsx]: We need to add it, I'll push the fix

Comment (Tbaut) [reply packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PlanDetails.tsx]: done in bb435a987f4f577bd2a0f9a3c0fd47e0c546fa90

**Review by Tbaut — APPROVED** [review]
Looks good if I didn't break anything :D

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PlanDetails.tsx (hunk 1 above)
Comment (asnaith) [reply packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PlanDetails.tsx]: @Tbaut Ah ok! I see what you did here. Thanks for that! I'll bear this in mind the next time placing an identifier directly on the element doesn't work.

**Review by FSM1 — APPROVED** [review]

**Pull request merged**